addon/globalPlugins/pcKbBrl.py

```diff
@@ -367,15 +411,111 @@ def makeSettings(self, settingsSizer):
 			self.cancelKeysEdit.SetValue(config.conf["pcKbBrl"]["cancelKeys"])
 		except KeyError:
 			pass
+		# Translators: label of a dialog.
+		dot1KeysLabel = _("Type the characters for &dot1 when writing with one hand.")
```

Comment:
Regarding the labels, various improvements in the following suggestion:
* Put the accelerator on the number of the dot so that you can use alt+1 to 8 as accelerator key
* Rephrase the label: usually we do not include the action to do in the label, e.g. we do not write "Check this checkbox to enable one hand typing", so I have removed "type".
* Punctuation: at the end of an option's label, use colon or nothing rather than period.

This is the suggestion for dot1 but you should make the same modification for dots 2 to 8 and for nullKey when applicable:

```suggestion
		dot1KeysLabel = _("Characters for dot&1 when writing with one hand:")
```